RickLeinecker/Fortuna
# 3v3 favorites APIs

**Description**
Made APIs for 3v3 favorites:
- `/api/tank/setFavoriteTankTeam`
  - x-auth-token in Header
  - Array of 3 MongoIds to be added in Body
  - Returns array of the 3 MongoIds or an error.
- `/api/tank/getFavoriteTankTeam`
  - x-auth-token in Header
  - Returns array of the 3 MongoIds or an error.
- `/api/tank/unfavoriteTankTeam`
  - x-auth-token in Header
  - Returns success prompt of setting favoriteTanks to an [] and wager to 0 or an error.

**Resolves These Issues**
Resolves #401 

**Type of change**
Check options that are relevant.


**Steps to Verify Functionality**
1. Logon and make 3 tanks.
2. On postman use the setFavoriteTankTeam route and confirm it worked on MongoDB Compass
3. On postman use the getFavoriteTankTeam route and confirm it returned things on postman
4. On postman use the unfavoriteTankTeam route and confirm it worked on MongoDB Compass

**Final Checklist:**
Go down the list and check them off.
Passes flow:
![image](https://user-images.githubusercontent.com/24559442/78921878-977b1b80-7a63-11ea-9d10-4e7e854201d6.png)
